releasenotes/notes/improve-systemd-config-a82bca0014f6ccc6.yaml

```diff
@@ -0,0 +1,19 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    Update configuration. Before if `unit_names` and `unit_regexes` are not set,
```

Comment:
I think the changelog is shared with every other PR for the agent, aka we should specify that all of this is about the systemd integration.

Reply:
fixed